specification/resources/resource-manager/readme.md

```diff
@@ -358,6 +349,15 @@ input-file:
 - Microsoft.Resources/stable/2015-11-01/resources.json
 ```
 
+### Tag: package-subscriptions-2019-11
+
+These settings apply only when `--tag=package-subscriptions-2019-11` is specified on the command line.
+
+``` yaml $(tag) == 'package-subscriptions-2019-11'
+input-file:
+- Microsoft.Resources/stable/2019-11-01/subscriptions.json
```

Comment:
Fix the [incorrect naming](https://github.com/Azure/azure-rest-api-specs/pull/7839/files/8668e09bf93fd5a9cbaf00980a6cf50cdb6dee33#diff-d474de52c790d7028fb81d4abf0b9618R64) of `package-2019-11` introduced in https://github.com/Azure/azure-rest-api-specs/pull/7839.